k4e-agent-sti.sh

```diff
@@ -79,9 +79,10 @@ cd /home/builder/r4e
 #----------------------------------------
```

Comment:
pay attention that you're pointing to an operator that running on helios05 in line 77